src/raytracer/getTgayReflectionLoss.m

```diff
@@ -59,8 +59,8 @@
 reflectionCoefficient = getReflectance(MaterialLibrary,arrayOfMaterials,multipath);
 % Calculate reflection loss. It is the sum (in dB) of the mean
 % of the Vertical and Horizontal reflectances for each reflection.
-reflectionLoss = prod(mean(reflectionCoefficient));
-reflectionLossdB = -20*log10(abs(reflectionLoss)); 
+reflectionLoss = prod(sqrt(mean(abs(reflectionCoefficient).^2))); 
```

Comment:
what about this update? I cannot understand how the reflection coefficient was computed, could you give me som more info?
Specifically, I'm struggling to understand what B_v and B_h are and the formula for the reflection coefficient.
Some references might help, also in the code documentation